tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala

```diff
@@ -145,11 +146,11 @@ class VarDictJavaEndToEnd(tumorBam: PathToBam,
     val bias               = new ShellCommand(biasScript.toString) with Pipe[Any,Any]
     val toVcf              = new ShellCommand(vcfScript.toString, "-N", tn, "-E", "-f", minimumAf.toString) with Pipe[Any,Vcf]
     val removeRefEqAltRows = new ShellCommand("awk", "{if ($1 ~ /^#/) print; else if ($4 != $5) print}") with Pipe[Vcf,Vcf]
-    val sortVcf            = new SortVcf(dict=Some(dict))
+    val sortVcf            = new SortVcf(in=tmpVcf, out=out, dict=Some(dict))
 
     bias.requires(Cores(0), Memory("32m"))
     toVcf.requires(Cores(0), Memory("32m"))
 
-    root ==> (vardict | bias | toVcf | removeRefEqAltRows | sortVcf > out).withName(this.getClass.getSimpleName)
+    root ==> (vardict | bias | toVcf | removeRefEqAltRows > tmpVcf).withName("VarDictJava") ==> sortVcf ==> new DeleteFiles(tmpVcf)
```

Comment:
On (2) - it was that even when I set `out=someVcf` on the pipeline, the pipeline was leaving it as `Io.Stdout` on `SortVcf` and using a `>` to push it into the file.  So I'm writing to a file, but `SortVcf` thinks it's writing to stdout.